tests/metrics/test_metric_definitions.py

```diff
@@ -283,6 +302,7 @@ def test_ndcg_at_k():
     predictions = data["prediction"]
     targets = data["target"]
     result = ndcg_at_k(3).eval_fn(predictions, targets)
+    assert ndcg_at_k(3)(predictions=predictions, targets=targets) == result
```

Comment:
Can we check the output of `help(ndcg_at_k)` is correct?

Reply:
<img width="670" alt="image" src="https://github.com/user-attachments/assets/a5f657cf-4dfb-4278-9232-0413b5912f6e">

Reply:
Can we add a test?

Reply:
Added!